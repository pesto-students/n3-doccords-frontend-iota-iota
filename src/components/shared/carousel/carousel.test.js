import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import MyCarousel from "components/shared/carousel/Carousel";

describe("MyCarousel", () => {
  const list = [
    {
      title: "Item 1",
    },
    {
      title: "Item 2",
    },
  ];

  it("rendering without Error", () => {
    const wrapper = shallow(<MyCarousel data={list} handleClick={() => {}} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
