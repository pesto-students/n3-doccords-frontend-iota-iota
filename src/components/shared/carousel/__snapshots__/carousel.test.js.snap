// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyCarousel rendering without Error 1`] = `
<div>
  <Carousel
    additionalTransfrom={0}
    arrows={true}
    autoPlaySpeed={1000}
    centerMode={false}
    className=""
    containerClass="carousel-container"
    customTransition="all .5"
    dotListClass="custom-dot-list-style"
    draggable={true}
    focusOnSelect={false}
    infinite={true}
    itemClass="carousel-item-padding-100-px"
    keyBoardControl={true}
    minimumTouchDrag={80}
    pauseOnHover={true}
    renderArrowsWhenDisabled={false}
    renderButtonGroupOutside={false}
    renderDotsOutside={false}
    responsive={
      Object {
        "desktop": Object {
          "breakpoint": Object {
            "max": 3000,
            "min": 1024,
          },
          "items": 4,
          "slidesToSlide": 1,
        },
        "mobile": Object {
          "breakpoint": Object {
            "max": 464,
            "min": 0,
          },
          "items": 1,
          "slidesToSlide": 1,
        },
        "tablet": Object {
          "breakpoint": Object {
            "max": 1024,
            "min": 464,
          },
          "items": 2,
          "slidesToSlide": 1,
        },
      }
    }
    showDots={false}
    sliderClass=""
    slidesToSlide={1}
    ssr={true}
    swipeable={true}
    transitionDuration={500}
  >
    <WithStyles(ForwardRef(GridListTile))
      key="0"
      onClick={[Function]}
      style={
        Object {
          "height": "300px",
          "maxWidth": "500px",
          "padding": "5px",
        }
      }
    >
      <img
        alt="Item 1"
      />
      <WithStyles(ForwardRef(GridListTileBar))
        title="Item 1"
      />
    </WithStyles(ForwardRef(GridListTile))>
    <WithStyles(ForwardRef(GridListTile))
      key="1"
      onClick={[Function]}
      style={
        Object {
          "height": "300px",
          "maxWidth": "500px",
          "padding": "5px",
        }
      }
    >
      <img
        alt="Item 2"
      />
      <WithStyles(ForwardRef(GridListTileBar))
        title="Item 2"
      />
    </WithStyles(ForwardRef(GridListTile))>
  </Carousel>
</div>
`;
